connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/streaming/StreamingQuery.scala

```diff
@@ -199,6 +205,26 @@ class RemoteStreamingQuery(
     // scalastyle:on println
   }
 
+  override def exception: Option[StreamingQueryException] = {
+    val exception = executeQueryCmd(_.setException(true)).getException
+    if (exception.hasExceptionMessage) {
+      // TODO(SPARK-43206): Add more information to StreamingQueryException.
+      Some(
+        new StreamingQueryException(
+          "",
+          cause = null,
+          "",
+          "",
+          errorClass = "STREAM_FAILED",
```

Comment:
Just curious, is the existing error framework accessible in Spark Connect jvm client?

Reply:
I believe it's not yet... The general error handling framework for jvm client is not ready yet. And they only include error message and stack trace so far. I'm thinking to implement our own proto message to also include the offsets, which will be added in next PR

Reply:
I see, thanks for the clarification!